chart/templates/uds-package.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: uds.dev/v1alpha1
+kind: Package
+metadata:
+  name: nginx
+  namespace: {{ .Release.Namespace }}
+spec:
+  # This will be broken until an authservice exclusion is added to uds-core
+  monitor:
+    - selector:
+        nginx-metrics: "true"
+      targetPort: 8081
+      portName: metrics
+      path: /metrics/
+      description: Metrics
+      podSelector:
+        app.kubernetes.io/name: nginx
+
+  sso:
+    - name: Nginx Authserviced
+      clientId: uds-package-nginx
+      redirectUris:
+        - https://nginx.uds.dev/login
```

Comment:
Should probably be a variable and ideally we could derive the base URL?

Reply:
Missed that, I'll change it to reference values.domain